.github/workflows/build.yml

```diff
@@ -1,6 +1,11 @@
 name: LAGraph CI
 
-on: [push, pull_request]
+on:
+  workflow_dispatch:
+  push:
+    branches-ignore:
+      - '**/*dev2'
```

Comment:
Why this ignore?

Reply:
Because I use the dev2 branch to push to directly, and I don't want to run tests all the time.  If I want to run the test automatically, I can do that with a PR to another branch, say '*dev'.

Reply:
Also, *dev2 branches can contain LAGraph versions that rely on GrB get/set, which are not in the CI and they break.